src/checks/pokerus.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  description: 'Illegal pokerus strain for Pokémon',
+  field: 'pokerusStrain',
+  test (pkmn) {
+    const strain = pkmn.pokerusStrain;
+    const duration = pkmn.pokerusDuration;
+
+    if (strain === 0) {
+      return duration === 0;
+    }
+    if (strain >= 0 || strain <= 15) {
```

Comment:
Using an `if` statement here is unnecessary because the condition will always be true.